assets/global.js

```diff
@@ -68,6 +68,13 @@ function trapFocus(container, elementToFocus = container) {
   document.addEventListener('focusin', trapFocusHandlers.focusin);
 
   elementToFocus.focus();
+
+  if (elementToFocus.tagName === 'INPUT' &&
+    ['search', 'text', 'email', 'url'].includes(elementToFocus.type) &&
+    elementToFocus.value) {
+    const end = elementToFocus.value.length;
+    elementToFocus.setSelectionRange(0, end);
```

Comment:
Do we need to create the const if it's only used once?
```suggestion
    elementToFocus.setSelectionRange(0, elementToFocus.value.length);
```